variables.mk

```diff
@@ -86,7 +86,7 @@ CHIPYARD_FIRRTL_DIR = $(base_dir)/tools/firrtl
 #########################################################################################
 # names of various files needed to compile and run things
 #########################################################################################
-long_name = $(MODEL_PACKAGE).$(MODEL).$(CONFIG)
+long_name = $(CONFIG_PACKAGE).$(MODEL).$(CONFIG)
```

Comment:
Random thought: Should we look to re-organize this into something like "MODEL.CONFIG_PACKAGE.CONFIG"? Just to have it be clear that the CONFIG comes from CONFIG_PACKAGE and the MODEL can be located somewhere else?

Reply:
Also, should we start deprecating the support for the Hwacha generator @colinschmidt? Is there a use case for it to still exist?

Reply:
I think that makes more sense.

Reply:
I'm in favor of removing `SUB_PROJECT`.

Reply:
Likewise at the beginning it made sense. Now less so.

Reply:
@colinschmidt Thoughts on the Hwacha generator? If you don't use it then we can just use the Chipyard generator and get rid of the "SUB_PROJECT=hwacha"

Reply:
It needs to provide feature parity.

Reply:
Unless i screwed up porting the Makefrag generation, it should now.

Reply:
Looks reasonable to me. I'll test after the merge and fix it if its broken.